index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="main">  
+            <a href="index.html" class="my-logo">My Logo</a>
+        
+            <nav class="my-menu">
+                <div></div>
+                <div></div>
+                <div></div>
+            </nav>
+
+    </header>
+    
+    <section class="headline">
+        <section class="head1">
+            <h1> I’m George
+                Glad to see you!</h1>
+        </section>
```

Comment:
- [ **OPTIONAL** ] as this wrap a block of a section, I suggest you use a `div` tag as a container instead of a `section` tag for good semantics tag i.e.
```HTML
        <div class="head1">
            <h1> I’m George
                Glad to see you!</h1>
        </div>
```